qpc/messages.py

```diff
@@ -207,6 +207,10 @@
     'No report detail available for scan job %s.'
 REPORT_NO_DETAIL_REPORT_FOR_REPORT_ID = \
     'No report detail available for report id %s.'
+REPORT_NO_INSIGHTS_REPORT_FOR_SJ = \
+    'No insights report available for scan job %s.'
+REPORT_NO_INSIGHTS_REPORT_FOR_REPORT_ID = \
+    'Report %s does not exist.'
```

Comment:
Should we specify insights here? Because it is possible for the other reports (details/deployments) to exist but not the insights report.

Reply:
yep